solutions/02-useLazyLoadQuery/src/App.tsx

```diff
@@ -9,8 +9,8 @@ import { AppQuery } from './__generated__/AppQuery.graphql';
 const App = () => {
   const response = useLazyLoadQuery<AppQuery>(
     graphql`
-      query AppQuery {
-        posts(first: 10) {
+      query AppQuery ($first: Number) {
+        posts(first: $first) {
```

Comment:
also try using a `defaultValue`

Reply:
Correct me if I'm wrong, but defaultValue is only available inside '@argumentDefinitions'